pytest.ini

```diff
@@ -7,6 +7,7 @@ markers =
     # Test types
     smoke: Mark tests as smoke tests.
     smoke_multi: Mark multi-cluster tests as smoke tests.
+    smoke_single: Mark single-cluster tests as smoke tests.
```

Comment:
I think it should be `smoke_single_cluster`, also for multi.

Reply:
done